source/process/sg_grammar-spelling-mechanics.rst

```diff
@@ -1,130 +1,130 @@
 
-Commas
-------
-
-As a general rule, the serial comma results in greater clarity. However, there are always edge cases where a serial comma adds confusion to a sentence. Therefore, the Mattermost documentation will use the following rule for commas:
+Tone
+~~~~~
 
-Use the serial comma unless doing so decreases clarity and understanding of the sentence.
+Use a direct, impartial tone. Mattermost documentation is written to answer questions and solve problems, not to entertain.
 
 Preferred
-  The cows ran from wolves, coyotes, and mosquitoes.
+  If login fails due to an invalid password, turn Caps Lock off and then attempt to re-enter your password. 
 
 Avoid
-  The cows ran from wolves, coyotes and mosquitoes.
+  Failed sign in? No problem! Simply enter the correct password and we'll let you in right away.
 
-Tone
-----
+Voice
+~~~~~~
 
-Use a direct, impartial tone. Most readers of the documentation are looking for answers and solutions to their problems; they are not looking for entertainment.
+Use active voice in preference to passive voice. Active voice has the subject of a sentence doing the action. In passive voice, the subject has an action done to it.
 
 Preferred
-  If your password is rejected, check to make sure that Caps Lock is off, and then carefully type it in again. 
+  The system opens the *Status* pane.
 
 Avoid
-  Failed sign in? No problem! Simply enter the correct password and we'll let you in right away.
+  The *Status* pane is will be opened by the system.
+  
+Person
+~~~~~~
 
-.. _capital:
+Use the second person and avoid the first person.
 
-Capitalization
---------------
+Preferred
+  View the status in the *Status* pane.
 
-Use title case for page titles and sentence case for section titles.
+Avoid
+  We'll view the status in the *Status* pane.
+  
+Tense
+~~~~~~
 
-Title case
-  Grammar, Spelling, and Mechanics
+Use the present tense.
 
-Sentence case
-  Language and spelling
+Preferred
+  Sharing this link lets other users view the linked message.
 
-Voice
------
+Avoid
+  Sharing this link will let other users view the linked message.
+  
+Commas
+------
 
-Use active voice in preference to passive voice. Active voice has the subject of a sentence doing the action. In passive voice, the subject has an action done to it. Use passive voice only when you want to emphasize the action more than the subject.
+In general, Mattermost documentation will use the serial comma unless doing so decreases clarity and understanding of the sentence.
 
 Preferred
-  The system opens the *Status* pane.
+  The cows ran from wolves, coyotes, and mosquitoes.
 
 Avoid
-  The *Status* pane is opened by the system.
+  The cows ran from wolves, coyotes and mosquitoes.
 
-Person
-------
+.. _capital:
 
-Use the second person and avoid the first person.
+Capitalization
+--------------
 
-Preferred
-  You can view the status in the *Status* pane.
+Use title case for page titles and section titles.
 
```

Comment:
I would say that almost all our documentation currently uses title case as the standard. The visual separator between page titles and section titles would be the size of the heading, when PM's discussed we thought that was sufficient. Are you okay with using title case as the standard for now and we can revisit if it comes up again in the future?